### ISSUE TITLE
Treating several alternative TS definition

### ISSUE THREAD

**Issue by olejandro** [etsap-TIMES/times-excel-reader#133]
Currently the tool fails if several alternative ts definitions are present.

**Comment by Antti-L** [comment]
@olejandro Does VEDA support several **alternative ts definitions**? I didn't know that; Is there an example of using such somewhere?  I know scenario-specific ~Timeslices tables, but I think only one table should be included in any model run?

**Comment by olejandro** [comment]
Thanks @Antti-L. Yes, I believe we are writing about the same issue. Tbh, I don't know what Veda would do if several  scenario files each with it's own ts definition were included in a scenario group...
The tool requires only one such definition to be present; it fails when there is more than one.
The tool already accepts passing a list of files to be processed (as an alternative to processing all the files in a folder). So maybe the tool should advise the use to use this functionality to include a single ts definition?

**Comment by Antti-L** [comment]
Yes, agree, I think it should advice the user about that funcionality if such an error occurs.

**Comment by olejandro** [comment]
On the other hand, it really depends on the use case. What I write above probably makes sense, if the user intends to run the model. It doesn't make sense if the intention of the user is to use the data in another UI, e.g. TIMES MIRO App that has scenario management capabilities. 🤔
